lib/coinbase/client/api/addresses_api.rb

```diff
@@ -6,7 +6,7 @@
 The version of the OpenAPI document: 0.0.1-alpha
 Contact: yuga.cohler@coinbase.com
 Generated by: https://openapi-generator.tech
-Generator version: 7.7.0
+Generator version: 7.6.0
```

Comment:
seeing a lot of this Generator change, is it intended?

Reply:
Yes, these are automated